<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>GameOverlay</class>
    <widget class="QWidget" name="GameOverlay">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>200</width>
                <height>200</height>
            </rect>
        </property>
        <layout class="QVBoxLayout" name="topLayout">
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <widget class="QWidget" name="OverlayBackground">
                    <property name="geometry">
                        <rect>
                            <x>0</x>
                            <y>0</y>
                            <width>200</width>
                            <height>200</height>
                        </rect>
                    </property>
                    <layout class="QVBoxLayout" name="mainLayout">
                        <item>
                            <spacer name="verticalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QLabel" name="overlayTitle">
                                <property name="text">
                                    <string>2048</string>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignCenter</set>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="overlayDescription">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="text">
                                    <string>2048</string>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignCenter</set>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="verticalSpacer_2">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QLabel" name="bottomText">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="text">
                                    <string>Press any key to close the game</string>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignCenter</set>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>